src/PIL/BmpImagePlugin.py

```diff
@@ -263,7 +263,7 @@ def _open(self):
         # read 14 bytes: magic number, filesize, reserved, header final offset
         head_data = self.fp.read(14)
         # choke if the file does not have the required magic bytes
-        if head_data[0:2] != b"BM":
+        if not _accept(head_data[0:2]):
```

Comment:
Just pass the data and let `_accept` do whatever it needs to? (Like in the other files.)

```suggestion
        if not _accept(head_data):
```

```python
def _accept(prefix):
    return prefix[:2] == b"BM"
```

Reply:
Yep. Missed this one.